public/locales/en/translation.json

```diff
@@ -193,7 +194,6 @@
             "label": "Run status:",
             "successful": "Successful",
             "running": "Running",
-            "dataingestioninprogress": "Transferring results",
```

Comment:
Why did  you remove this label ? Maybe it was a failed merge or rebase, could you check it please ?